src/context/Archive-Context.js

```diff
@@ -0,0 +1,20 @@
+import React,{createContext,useContext,useState} from "react"
+
+const ArchiveContext = createContext()
+
+const ArchiveProvider = ({children}) => {
+    const [archiveNotes, setArchiveNote] = useState([])
+
+    const addToArchive = (note) => setArchiveNote(prev => [...prev,note])
+
+    
```

Comment:
remove spacing

Reply:
Space is given only for readability purposes